blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -1,23 +1,81 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { useFusionContext } from "fusion:context";
```

Comment:
We are also missing the bandito support that promo's has, you'll need `useComponentContext`

and the following code within the block

```
const { registerSuccessEvent } = useComponentContext();
```

as well as the `onClick` event

```
onClick={registerSuccessEvent}
```

Reply:
Thanks! Will add.

Reply:
@matthewroach : Added. Thanks!